src/linear_algebra/bilinear_form.lean

```diff
@@ -1016,6 +1016,16 @@ begin
   exact H1,
 end
 
+lemma is_refl (H : is_alt B₁) : refl_bilin_form.is_refl B₁ :=
+begin
+  intros x y h,
+  rw [←neg H, h],
+  simp,
+end
+
+lemma ortho_sym (H : is_alt B₁) {x y : M₁} :
+  is_ortho B₁ x y ↔ is_ortho B₁ y x := refl_bilin_form.ortho_sym (is_refl H)
```

Comment:
Note if we made the rename I suggest in another PR, this will become the shorter
```lean
  is_ortho B₁ x y ↔ is_ortho B₁ y x := H.is_refl.ortho_sym 
```